### PR TITLE
Add licensing #20

### PR DESCRIPTION
**Pull request by ChiuchiuSorin**
* Added licensing dependencies
* Added license verifications for WebHOme and templates
* Applied xar:format

### CODE REVIEW

**Comment by ChiuchiuSorin** [comment]
Error message when there is a missing license:
image
Error message when there is a call for an individual template without a license:
image

File: application-admintools-ui/src/main/resources/AdminTools/WebHome.xml [hunk 1]
```diff
@@ -34,7 +36,18 @@
...
+  <content>{{include reference="Licenses.Code.VelocityMacros"/}}
+
+{{velocity}}
+#set ($mainReference = $services.model.createDocumentReference('', ['AdminTools', 'Code'], 'ConfigurationClass'))
+#if ($services.licensing.licensor.hasLicensureForEntity($mainReference))
+  {{dashboard/}}
+#else
+{{error}}
+  #getMissingLicenseMessage('adminTools.extension.name')
+{{/error}}
```
Comment (oanalavinia) [inline application-admintools-ui/src/main/resources/AdminTools/WebHome.xml:48]: Indentation missing

**Review by oanalavinia — COMMENTED** [review]
Just one comment

File: application-admintools-default/src/main/resources/templates/securityTemplate.vm [hunk 2]
```diff
@@ -17,34 +17,43 @@
...
+      </ul>
+    </div>
+  #else
+    #set($warningMessage = $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.error')))
+    #warning($warningMessage)
+  #end
 #else
-  #set($warningMessage = $escapetool.xml($services.localization.render('adminTools.dashboard.section.security.error')))
-  #warning($warningMessage)
+  #includeMacros('Licenses.Code.VelocityMacros')
```
Comment (mflorea) [inline application-admintools-default/src/main/resources/templates/securityTemplate.vm:55]: Do you need to include it twice? You also include it at the top of the template.

**Review by mflorea — COMMENTED** [review]
Looks good with only one small comment.

File: application-admintools-default/src/main/resources/templates/securityTemplate.vm (hunk 2 above)
Comment (ChiuchiuSorin) [reply application-admintools-default/src/main/resources/templates/securityTemplate.vm:55]: Missed that field. I removed the first call for the macro.

**Pull request merged**